charts/collabora-online/0.0.1/questions.yaml

```diff
@@ -0,0 +1,314 @@
+          description: 'Use backslash "\" before dots ".". Use pipe "|" to separate multiple domains'
+          schema:
+            type: string
+            default: 'nextcloud\.domain\.tld|othernextcloud\.domain\.tld'
+            required: true
+        - variable: username 
+          label: "Username for WebUI"
+          schema:
+            type: string
+            default: "admin"
+            required: true
+        - variable: password
+          label: "Password for WebUI"
+          schema:
+            type: string
+            private: true
+            default: "changeme"
+            required: true
+        - variable: dictionaries
+          label: "Dictionaries to use, leave empty to use all"
+          schema:
+            type: string
+            default: "de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru"
+        - variable: extra_params
+          label: "Extra Parameters to add"
+          description: 'e.g. "–o:welcome.enable=false", See more on /etc/loolwsd/loowsd.xml. Separate params with space'
+          schema:
+            type: string
+            default: "-o:welcome.enable=false -o:user_interface.mode=notebookbar"        
+        - variable: server_name
+          label: "Server Name"
+          description: "When this environment variable is set (is not “”), then its value will be used as server name in /etc/loolwsd/loolwsd.xml. Without this, CODE is not delivering a correct host for the websocket connection in case of a proxy in front of it."    
+          schema:
+            type: string
+            default: 'collabora\.domain\.tld'
+        
+  # Storage
+  # Configure app volumes
+  - variable: appVolumeMounts
+    label: "app storage"
+    group: "Storage"    
+    schema:
+      type: dict
+      attrs:
+        # Config ------------------------
+        - variable: config
+          label: "Config Volume"
+          schema:
+            type: dict
+            $ref:
+              - "normalize/ixVolume"
+            attrs:
+              - variable: enabled
+                label: "Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+                  hidden: true
+                  editable: false
+              - variable: emptyDir
+                label: "emptyDir"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: datasetName
+                label: "Name"
+                schema:
+                  type: string
+                  default: "config"
+                  required: true
+                  editable: false
+              - variable: mountPath
+                label: "Mount path inside the container"
+                schema:
+                  type: path
+                  required: true
+                  default: "/etc/loolwsd"
```

Comment:
are you sure there is no environment variable to move this to /config?
in case there is, thats prefered.

Reply:
Neither an env var or extra param to set a different path for /etc/loolswd/loolswd.xml
it only have options to move every other file in /etc/loolwsd, (certs, keys, ca) extect the config file (xml)

Reply:
Thats what we call "config":  the config folder, not the files.

Reply:
Yes, but it only allows to change path of specific files but not folder. e.g.
`<key_file_path desc="Path to the key file" relative="false">/etc/loolwsd/key.pem</key_file_path>`
You can't change the `/etc/loolwsd` which also contains the `loolswd.xml` (the config file)

Reply:
ahh shame.

Reply:
Yea... but, even if the `/etc/loolwsd` is not persistance, It's not a problem. The whole config file it's generated on creation/update, and changes to it can be made with the `extra_params` field
So if it isn't necessary to have storage options, we can remove them.